operators/azure/cloud_base_kernel.hpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ocos.h"
+#include "gsl/span"
+
+namespace ort_extensions {
+
+/// <summary>
+/// Base kernel for custom ops that call cloud endpoints.
+/// </summary>
+class CloudBaseKernel : public BaseKernel {
+ protected:
+  CloudBaseKernel(const OrtApi& api, const OrtKernelInfo& info);
+  virtual ~CloudBaseKernel() = default;
+
+  // Names of attributes the custom operator provides.
+  static constexpr const char* kUri = "model_uri";           // required
+  static constexpr const char* kModelName = "model_name";    // optional
+  static constexpr const char* kModelVer = "model_version";  // optional
+  static constexpr const char* kVerbose = "verbose";
+
+  static constexpr int MinimumSupportedOrtVersion = 14;
```

Comment:
nit: naming convention
```suggestion
  static constexpr int kMinimumSupportedOrtVersion = 14;
```